automation_tools/__init__.py

```diff
@@ -573,31 +573,69 @@ def install_puppet_scap_client():
     run('yum -y install puppet-foreman_scap_client', warn_only=True)
 
 
-def setup_foreman_discovery():
-    """Task to setup foreman discovery."""
-    # Install required packages to enable discovery plugin
+def setup_foreman_discovery(sat_version, upstream=False):
+    """Task to setup foreman discovery.
+
```

Comment:
Let's add the ENV_VAR required for `setup_foreman_discovery`, similar to `setup_ddns` method, it would help us quickly verify the required ones.
`The following environment variables affect this command:
  * `PXE_DEFAULT_TEMPLATE_URL_FOR_70`
  * `PXELINUX_DISCOVERY_SNIPPET_URL`

I know it's not available for all the methods, but I feel it is better to have this.